feat: helm global values updates (#16062)
grafana/loki: docs/sources/setup/install/helm/reference.md

			<td><pre lang="json">
"kube-dns"
</pre>
</td>
		</tr>
		<tr>
			<td>global.extraArgs</td>
			<td>list</td>

> 			<td>Common additional CLI arguments for all jobs (i.e. -log.level debug, -config.expand-env=true or -log-config-reverse-order) scope: admin-api, backend, bloom-builder, bloom-gateway, bloom-planner, compactor, distributor, index-gateway, ingester, overrides-exporter, pattern-ingester, querier, query-frontend, query-scheduler, read, ruler, write</td>

Reviewer: ```suggestion
			<td>Common additional CLI arguments for all jobs (that is, -log.level debug, -config.expand-env=true or -log-config-reverse-order) scope: admin-api, backend, bloom-builder, bloom-gateway, bloom-planner, compactor, distributor, index-gateway, ingester, overrides-exporter, pattern-ingester, querier, query-frontend, query-scheduler, read, ruler, write</td>
```
